### DISCUSSION TITLE
Add option to add legal links to the frontpage

### DISCUSSION THREAD

**Discussion by github-luckow-org** [postalserver/postal#2423] [Feature suggestions]
Maybe an edge case.
I use postal on a dedicated domain (no other software is involved except postal) - and I have delivery problems with one of the biggest mail providers in Germany – T-Online.
Getting in touch with their support team, I got the response that the issue is the missing contact address on the domain itself. 
[https://postmaster.t-online.de/index.en.html](https://postmaster.t-online.de/index.en.html) (4.1 Requirements for smooth access to our e-mail exchanges)

The easiest way for me should be something like adding legal/contact links to the register/login portal. But there is no option in the backend. Instead of this missing feature, I have to figure out how to redirect parts of the domain with Caddy or make some magic with the docker files. 

My suggestion is: add configuration options in the backend to add some links to the postal "frontpage".

**Comment by willpower232** [comment]
Are you not able to add this text in whatever is sending the messages? Postals primary responsibility is sending messages (and you can see there are already issues adding tracking pixels to most emails so adding links is likely to be even more challenging)

**Reply by uhlhosting** [reply]
@willpower232 I believe he refers to link in postal interface not in the mails themselves. That would imply that tmobile already have the mail have checked it for links and i doubt thats even in this context, since from my understanding they might reject mail. I believe where he points is that having postal acting as a sending mail server. Some providers mighth want to see the legal terms of who is behing that mail servers. Its how I understood that what was said.

**Reply by willpower232** [reply]
ah good point, I am not very awake apparently!

I'm not sure why links would need to be present on Postals UI though as that is internal to the organisation, if there are links to be shown they should be on the public domain you're sending from i.e. your own website surely?
